File: maxhanna.client/src/services/datacontracts/nexus/unit-upgrade-stats.ts
export class UnitUpgradeStats {
  unitLevel: number;
  damageMultiplier: number;
  duration: number;

  constructor(unitLevel: number, damageMultiplier: number, duration: number) {
    this.unitLevel = unitLevel;
    this.damageMultiplier = damageMultiplier;
    this.duration = duration;
  }
}

File: maxhanna.Server/Controllers/DataContracts/Nexus/UnitUpgradeStats.cs
namespace maxhanna.Server.Controllers.DataContracts.Nexus
{
	public class UnitUpgradeStats
	{
		public int UnitLevel { get; set; }
		public decimal DamageMultiplier { get; set; }
		public int Duration { get; set; }

		public UnitUpgradeStats()
		{
		}

		public UnitUpgradeStats(int unitLevel, decimal damageMultiplier, int duration)
		{
			UnitLevel = unitLevel;
			DamageMultiplier = damageMultiplier;
			Duration = duration;
		}
	}
}